src/+otp/+kortewegdevries/+presets/Canonical.m

```diff
@@ -0,0 +1,47 @@
+classdef Canonical < otp.kortewegdevries.KortewegdeVriesProblem
+    methods
+        function obj = Canonical(varargin)
+
+            Domain = [-10, 10];
+            Nx = 200;
+            theta = 0;
+            nu = -1;
+            alpha = -3;
+            rho = 0;
+            uinit = @(x) 6*(sech(x).^2);
+
+            p = inputParser;
+            addParameter(p, 'Domain', Domain);
+            addParameter(p, 'Nx', Nx);
+            addParameter(p, 'InitialCondition', uinit);
+            addParameter(p, 'Theta', theta);
+            addParameter(p, 'Alpha', alpha);
+            addParameter(p, 'Nu', nu);
+            addParameter(p, 'Rho', rho);
+
+            parse(p, varargin{:});
+            
+            s = p.Results;
+            
+            params = otp.kortewegdevries.KortewegdeVriesParameters;
+            params.Domain = s.Domain;
+            params.Nx = s.Nx;
+            params.InitialCondition = s.InitialCondition;
```

Comment:
Different initial conditions result in in chaotic and non chaotic problems. I get that you want each initial condition to be baked into different presets, but there are infinitely many such interesting initial conditions. I'd like the user to be able to set whatever initial condition inside here. Do you prefer to have the user set their initial condition by changing model.Y0 in their program?

Reply:
The initial condition can still be passed to `Canonical`. If a user wants to modify it, it should be with `Y0`. Otherwise, it's duplicated and can become inconsistent